code/modules/ninja/suit/suit_initialisation.dm

```diff
@@ -1,92 +1,94 @@
+	addtimer(CALLBACK(src, .proc/ninitialize_seven, delay, U), delay)
 
-			sleep(10)
-			to_chat(usr, "<span style='color: #0000ff;'>VOID-shift device status: <b>ONLINE</b>.\nCLOAK-tech device status:<b>ONLINE</b></span>")
+/obj/item/clothing/suit/space/space_ninja/proc/ninitialize_seven(delay, mob/living/carbon/human/U)
+	to_chat(U, "<span class='notice'>All systems operational. Welcome to <B>SpiderOS</B>, [U.real_name].</span>")
+	s_initialized = TRUE
+	ntick()
+	s_busy = FALSE
 
-			sleep(5)
-			to_chat(usr, "<span style='color: #0000ff;'>Primary system status: <b>ONLINE</b>.\nBackup system status: <b>ONLINE</b>.</span>")
-			if(suitCell)
-				to_chat(usr, "<span style='color: #0000ff;'>Current energy capacity: <b>[suitCell.charge]/[suitCell.maxcharge]</b>.</span>")
 
-			sleep(10)
-			to_chat(usr, "<span style='color: #0000ff;'>All systems operational. Welcome to <b>SpiderOS</b>, [usr.real_name].</span>")
-			//TODO: Grant ninja verbs here.
-			suitBusy = 0
-			suitActive = 1
 
-		else
-			suitBusy = 0
-			to_chat(usr, "<span style='color: #0000ff;'><b>NOTICE: </b>Suit de-activation protocals aborted.</span>")
-	else
-		to_chat(usr, "<span style='color: #ff0000;'><b>FĆAL �Rr�R</b>: ŧer nt recgnized, c-cntr-r䣧-ç äcked.")
-		return
+/obj/item/clothing/suit/space/space_ninja/proc/deinitialize(delay = s_delay)
+	if(affecting==loc)
+		var/mob/living/carbon/human/U = affecting
+		if(alert("Are you certain you wish to remove the suit? This will take time and remove all abilities.",,"Yes","No")=="No")
+			return
+		s_busy = TRUE
+		addtimer(CALLBACK(src, .proc/deinitialize_two, delay, U), delay)
+
+/obj/item/clothing/suit/space/space_ninja/proc/deinitialize_two(delay, mob/living/carbon/human/U)
+	to_chat(U, "<span class='notice'>Now de-initializing...</span>")
+	addtimer(CALLBACK(src, .proc/deinitialize_three, delay, U), delay)
+
+/obj/item/clothing/suit/space/space_ninja/proc/deinitialize_three(delay, mob/living/carbon/human/U)
+	to_chat(U, "<span class='notice'>Logging off, [U.real_name]. Shutting down <B>SpiderOS</B>.</span>")
+	addtimer(CALLBACK(src, .proc/deinitialize_four, delay, U), delay)
+
+/obj/item/clothing/suit/space/space_ninja/proc/deinitialize_four(delay, mob/living/carbon/human/U)
+	to_chat(U, "<span class='notice'>Primary system status: <B>OFFLINE</B>.\nBackup system status: <B>OFFLINE</B>.</span>")
+	addtimer(CALLBACK(src, .proc/deinitialize_five, delay, U), delay)
+
+/obj/item/clothing/suit/space/space_ninja/proc/deinitialize_five(delay, mob/living/carbon/human/U)
+	to_chat(U, "<span class='notice'>VOID-shift device status: <B>OFFLINE</B>.\nCLOAK-tech device status: <B>OFFLINE</B>.</span>")
+	cancel_stealth()//Shutdowns stealth.
+	addtimer(CALLBACK(src, .proc/deinitialize_six, delay, U), delay)
+
+/obj/item/clothing/suit/space/space_ninja/proc/deinitialize_six(delay, mob/living/carbon/human/U)
+	to_chat(U, "<span class='notice'>Disconnecting neural-net interface...</span>\green<B>Success</B><span class='notice'>.</span>")
```

Comment:
the "success" is supposed to be green, but isn't.

Reply:
No color macros allowed, use `font style` or a span class.